src/pages/tasks/NewTaskPage.js

```diff
@@ -153,7 +153,7 @@ function NewTaskPage(props) {
                     <View style={styles.mb5}>
                         <MenuItemWithTopDescription
                             description={props.translate('task.title')}
-                            title={props.task.title || ''}
+                            title={"ji" || ''}
```

Comment:
Hmm, seems like a mistake? Can you please revert

Reply:
Yeah , I will revert it